LLAP/llap_restart.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
```

Comment:
Can we add a step to copy this file into the master node of the cluster, and mention it in the readme so that user can just trigger it from there?

Reply:
good suggestion, I'll add an upload step for this file